pkg/arvo/lib/tapp.hoon

```diff
@@ -254,10 +254,10 @@
       ::  XX use only for development may break contracts!
       ::  XX if active clam contracts only to abort transaction?
       ::
-      ::  ~&  [%tapp-reset dap.bowl]
-      :: `this-tapp
-      ~|  [%tapp-load-incompatible dap.bowl]
-      !!
+      ~&  [%tapp-reset dap.bowl]
+      `this-tapp
+      ::  ~|  [%tapp-load-incompatible dap.bowl]
+      ::  !!
```

Comment:
I think this should remain a crash on `master`. It can be disabled for development if necessary, but we don't want to accidentally OTA with the state-discarding reset. Tapps that need to migrate state should be able to use `$&`.

Reply:
Ah, yes.